src/views/AnnualReport.vue

```diff
@@ -66,74 +66,74 @@
             <article
               v-if="isCoop"
               class="annual-report-article"
-              :class="agmDate ? 'agm-date-selected' : 'no-agm-date-selected'"
             >
               <!-- Page Title -->
               <header>
                 <h1 id="AR-header">File {{ARFilingYear}} Annual Report
                   <span class="font-italic" v-if="reportState"> &mdash; {{reportState}}</span>
                 </h1>
-                <p>Please verify or change your Office Addresses and Directors.</p>
+                <p>Select your Annual General Meeting date, then verify or change your Office Addresses
+                  and Directors</p>
               </header>
 
-              <template v-if="isAnnualReportEditable">
```

Comment:
I replaced this with `isCurrentFilingEditable` (global getter) on the action buttons.